.travis.yml

```diff
@@ -18,16 +19,22 @@ gemfile:
   - gemfiles/rails_4_2.gemfile
 matrix:
   exclude:
-    - rvm: 2.6.5
+    - rvm: 2.7.1
       gemfile: gemfiles/rails_4_2.gemfile
-    - rvm: 2.4.9
+    - rvm: 2.6.6
+      gemfile: gemfiles/rails_4_2.gemfile
+    - rvm: 2.4.10
       gemfile: gemfiles/rails_6_0.gemfile
-cache: bundler
+cache:
+  directories:
+    - vendor/bundle
```

Comment:
Due to our use of Appraisals, Bundler wasn't being cached correctly on Travis — now it will :)